helm/hpcc/templates/_helpers.tpl

```diff
@@ -579,3 +579,31 @@ Generate list of available services
   public: {{ $esp.public }}
 {{ end -}}
 {{- end -}}
+
+{{/*
+Add resource object
+Pass in a dictionary with me defined
+*/}}
+{{- define "hpcc.addResources" }}
+{{- if .me  }}
+resources:
+  requests:
+{{ toYaml .me | indent 4 }}
+  limits:
+{{ toYaml .me | indent 4 }}
+{{- end }}
+{{- end -}}
+
+{{/*
+Add resources object for stub pods
+Pass in a dictionary with me defined
+*/}}
+{{- define "hpcc.addStubResources" }}
+resources:
+  requests:
+    cpu: "50m"
+    memory: "50M"
+  limits:
+    cpu: "50m"
+    memory: "50M"
```

Comment:
I was testing this yesterday, and at least for the stubs that fork and call kubectl, 50M is not enough.
kubectl ends up getting killed by cgroup oom policy.
In my tests it looks like kubectl can consume about 28M.

I think changing this to 100M should be enough.

Reply:
We don't need to generate 'requests' if they are always going to be same as limits as they are here.
If limits, but no requests are specified, the requests are implicitly the same as the limits.